### ISSUE TITLE
traefik dashboard and whoami.docker.localhost not working

### ISSUE THREAD

**Issue by ghost** [56kcloud/traefik-training#2]
**Describe the bug**
traefik dashboard and whoami.docker.localhost not working

**To Reproduce**
1. 'run current 02-Configure-Traefik/docker-compose.file.yml'
2. try opening dashboard
3. will not open

**Expected behavior**
Add insecure=true in traefik.yml to make dashboard working

but still whoami.docker.localhost is giving 404

**Comment by jdaku** [comment]
Yes, its not working, even catapp.localhost on chapter-2 also not working;

**Comment by simsrw73** [comment]
> **Describe the bug** traefik dashboard and whoami.docker.localhost not working
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. 'run current 02-Configure-Traefik/docker-compose.file.yml'
> 2. try opening dashboard
> 3. will not open
> 
> **Expected behavior** Add insecure=true in traefik.yml to make dashboard working
> 
> but still whoami.docker.localhost is giving 404

This wasted a lot of my learning session, and more, trying to figure out. 

In "traefik.yml", swarmMode should NOT be used. I have no clue why it is in the file. I don't think we've been instructed to enable Docker Swarm and the Traefik docs seem to indicate that swarmMode requires a very different way of constructing the docker compose file when it is used. I hope it's covered at some point because I want to use it. But it should NOT be enabled here. It breaks everything.

**Comment by simsrw73** [comment]
> **Describe the bug** traefik dashboard and whoami.docker.localhost not working
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. 'run current 02-Configure-Traefik/docker-compose.file.yml'
> 2. try opening dashboard
> 3. will not open
> 
> **Expected behavior** Add insecure=true in traefik.yml to make dashboard working
> 
> but still whoami.docker.localhost is giving 404

In my previous (delete) reply, I forgot one of the changes I had made after I reset. Here is what I posted in the Q&A in the course:

 I got one variation to work but some others I tried, which I expected to work, did not work. I obviously still don't know what I'm doing.

Two things: 1) swarmMode wants a different configuration that what is supplied, and exposedByDefault: false requires the service to be enabled.

The config that works is traefik.yml: set swarmMode: false. And docker-compose.file.yml: add under whoami/label and new entry "- traefik.enable=true"
